project/src/main/world/creature/creature.gd

```diff
@@ -151,6 +151,15 @@ func _physics_process(delta: float) -> void:
 	_maybe_play_bonk_sound(old_non_iso_velocity)
 
 
+"""
+Increases the collision shape size for fatter creatures.
+
+This is not done on all creatures, because having fatter creatures collide with chairs/tables differently as they gain weight introduces problems. It's easier to leave their collision shapes consistent and just have a few visual bugs here and there.
```

Comment:
This line is much longer than 120 characters. Why did the delint script not catch this?

Reply:
The 'long lines' clause in the delinter was not running properly. I've fixed it and shortened this line.